Shared/mods/deathmatch/logic/CScriptDebugging.cpp

```diff
@@ -55,7 +56,7 @@ void CScriptDebugging::LogCustom(lua_State* luaVM, unsigned char ucRed, unsigned
     VSNPRINTF(szBuffer, MAX_STRING_LENGTH, szFormat, marker);
     va_end(marker);
 
-    LogString("", GetLuaDebugInfo(luaVM), szBuffer, 0, ucRed, ucGreen, ucBlue);
+    LogString("", GetLuaDebugInfo(luaVM), szBuffer, 0, ucRed, ucGreen, ucBlue, bOmitDebugInfo);
```

Comment:
What about not changing the `LogString` method definition and sending `nullptr` as the 2nd argument if `bOmitDebugInfo` is `true` ?
I mean, what's the point of getting infos from the LuaVM and sending it if you already know you will just discard that data ?
(Ofc, you will need to check for the `luaDebugInfo` before using it)